android/gradle.properties

```diff
@@ -12,10 +12,8 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryErro
 # org.gradle.parallel=true
 #Tue Dec 19 15:08:27 EST 2023
 KlaviyoReactNativeSdk_klaviyoAndroidSdkVersion=3.0.3
-KlaviyoReactNativeSdk_compileSdkVersion=31
-KlaviyoReactNativeSdk_kotlinVersion=1.8.0
+KlaviyoReactNativeSdk_kotlinVersion=1.7.0
 KlaviyoReactNativeSdk_minSdkVersion=23
-KlaviyoReactNativeSdk_ndkversion=21.4.7075529
 KlaviyoReactNativeSdk_targetSdkVersion=31
-android.enableJetifier=true
-android.useAndroidX=true
```

Comment:
are these defaulted to true now? or incompatible with new arch

Reply:
I was just noticing this as well. I'm not sure about jetifier, but we use androidX so that probably needs to come back

Reply:
Looking back in git, it appears I added them pretty early on, but I'm not clear why. They weren't part of the original template, nor are they in the latest one. And I'm not sure why we'd need them, since the SDK's android files do not make use of androidx at all. 
The example app does have `useAndroidx=true` which makes sense because it has androidx dependency for the app and such. 
So I'm thinking this is safe to remove.